test/utils/wasm3_engine.cpp

```diff
@@ -34,6 +35,18 @@ class Wasm3Engine : public WasmEngine
     Result execute(FuncRef func_ref, const std::vector<uint64_t>& args) final;
 };
 
+namespace
+{
+const void* env_adler32(IM3Runtime /*runtime*/, uint64_t* stack, void* mem)
+{
+    uint64_t* ret = stack;
+    uint32_t offset = *(uint32_t*)stack++;
```

Comment:
```suggestion
    const auto offset = *static_cast<uint32_t*>(stack++);
```

Reply:
Will do, it was copy paste from wasm3 :)